### ISSUE TITLE
Add section on editors/IDEAs

### ISSUE THREAD

**Issue by simeg** [sb2nov/mac-setup#154]
### I'm Submitting a ...
```
[ ] Bug report
[x] Tool/language/etc documentation request
```

### Tool/Language/etc
It would be awesome with a section of popular editors and IDEAs. This could include everything from emacs to IntelliJ and each section could have information about plugins etc.

Thoughts?

**Comment by Kyslik** [comment]
How many editors is there? I guess if we focus only on first "5" we can stick to current approach (each has its own section just like Vim and Sublime).

JetBrains IDEs are commercial (in a way you can not use it for free like Sublime with nag) except educational license, which is for free.

I mainly use, Visual Studio Code, Sublime, JetBrains (PyCharm + PHPStorm) and of course Vim on server side. I have tried Atom but VSC is better and I don't know any other editors :rofl:.

**Comment by simeg** [comment]
@Kyslik Yes, that's a good idea. Maybe this guide is not the proper place to tell people about Emacs ;) Let's stick with the top ~5 popular.

We could then have
* IntelliJ Community Edition
* Visual Studio Code
* Atom
* Sublime (this is already done, as you said)
* Vim (this is already done, as you said)

I can do IntelliJ and Atom, could you write a section on Visual Studio Code then @Kyslik? (I have never used it)

**Comment by Kyslik** [comment]
Yes, I am planning on doing some work for this repo since I am still on Yosemite and soon (not a clue when) I will upgrade to the latest, and when that comes I will write about PHP (fix for #56) and many more things that may include VSC.

Setting up VSC, just like with Sublime, strongly depends on what is person working with, is it a JavaScript user, maybe Python girl etc. I may cover my field, Python, JavaScript, PHP and markdown.

**Comment by sb2nov** [comment]
Related to #143, tagging so the issues get linked
